src/tag_propagation.h

```diff
@@ -0,0 +1,39 @@
+#ifndef DD_OPENTRACING_TAG_PROPAGATION_H
+#define DD_OPENTRACING_TAG_PROPAGATION_H
+
+// Some span tags are associated with the entire local trace, rather than just
+// a single span within the trace.  These tags are added to the local root span
+// before the trace is flushed.
+//
+// Among these root span tags, some are also propagated as trace context.
+// Propagated tags are packaged into the "x-datadog-tags" header in a
+// particular format (see the cpp file for a description of the format).
+//
+// This component provides serialization and deserialization routines for the
+// "x-datadog-tags" header format.
+
+#include <opentracing/string_view.h>
+
+#include <string>
+#include <unordered_map>
+
+namespace ot = opentracing;
+
+namespace datadog {
+namespace opentracing {
+
+// The names of designated trace-wide tags all have the same prefix, "_dd.p.".
+extern const ot::string_view trace_tag_prefix;
```

Comment:
Decide whether the shared prefix is at all relevant.

If not (which is how this is written now), then remove it.